scripts/generate_json_terms.py

```diff
@@ -6,14 +6,18 @@
 import xml.etree.ElementTree as ET
 
 def check_scope(domain, id):
-    if domain == 'facility':
+    if domain == 'facility' and id not in event_facilities:
         return ['events', 'places']
     if domain == 'theme':
         return ['events']
     if id in place_types:
         return ['places']
     return ['events']
 
+event_facilities = [
+    'H28fcfRKFQAQs00K9NF9hh', # Prikkelarm aanbod
+    '4Vz9eZf0cnQmtfqcGGnNMF', # Afspraken en voorspelbaarheid
```

Comment:
Aren't there more facilities that are exclusive to events then these two?
In UDB3 we allow these 23 facilities on events: https://github.com/cultuurnet/udb3-backend/blob/master/src/Event/EventFacilityResolver.php
But out of those 23 we only allow these 4 facilities on places: https://github.com/cultuurnet/udb3-backend/blob/master/src/Place/PlaceFacilityResolver.php

Reply:
Indeed, the JIRA call only mentioned to do it for the 2 new ones, but it's better to fix them all at once.

Reply:
Thanks for the extra fixes!